ci/gpu/build.sh

```diff
@@ -22,6 +22,7 @@ export PATH=/conda/bin:/usr/local/cuda/bin:$PATH
 export PARALLEL_LEVEL=4
 export CUDA_REL=${CUDA_VERSION%.*}
 export CUDF_HOME="${WORKSPACE}/cudf"
+export CUSPATIAL_HOME="${WORKSPACE}"
```

Comment:
How can CUSPATIAL_HOME be the parent directory of CUDF_HOME?

Reply:
I agree it's odd, it happens a few lines below:
https://github.com/rapidsai/cuspatial/blob/2a14919e70b9dd4ff90d0ea1e95d7423ec1699db/ci/gpu/build.sh#L54-L60